src/simulations/Simulations.jl

```diff
@@ -146,13 +147,23 @@ function LandSimulation(
 )
     # Enforce `h_atmos >= h_canopy` for models with canopy
     if hasproperty(model, :canopy)
-        h_atmos = model.canopy.boundary_conditions.atmos.h
-        h_canopy = model.canopy.hydraulics.compartment_surfaces[end]
-        @assert h_atmos >= h_canopy "Atmospheric height must be greater than or equal to canopy height"
+        # `atmos.h` is not meaningful for a CoupledAtmosphere, so we skip the check
+        if !(
+            model.canopy.boundary_conditions.atmos isa
+            ClimaLand.CoupledAtmosphere
+        )
+            h_atmos_min =
```

Comment:
is atmos.h a field? I guess in coupled sims it is

Reply:
But then this check is not carried out in coupled sims?

Reply:
Yes that's right. Float in standalone runs, and Field in coupled runs